synthtool/gcp/templates/java_library/.github/workflows/auto-release.yaml

```diff
@@ -16,8 +31,8 @@ jobs:
             return;
           }
 
-          // only approve PRs like "chore: release <release version>"
-          if ( !context.payload.pull_request.title.startsWith("chore: release") ) {
+          // only approve PRs like "chore(main): release <release version>"
+          if (  !context.payload.pull_request.title.startsWith("chore(main): release") ) {
```

Comment:
I presume you don't want to autorelease backported patches?

Reply:
Corrrect.